src/DIRAC/Core/Utilities/test/Test_Encode.py

```diff
@@ -27,6 +27,11 @@
 from pytest import mark, approx, raises, fixture, skip
 parametrize = mark.parametrize
 
+# function_scoped_fixture is only used in Python 3 compatible release of hypothesis
```

Comment:
Actually for that specific case, I think we could convert it to a module or scope fixture. It does not really need to change at each function. We could run all the diset test, then all the json, etc

Reply:
I don't think we can as the environment variables need to be different between different functions in the same module as they're parametrised?

Reply:
Indeed not, I thought we could run it differently